nesta/core/routines/arxiv/arxiv_grid_task.py

```diff
@@ -189,3 +190,30 @@ def run(self):
         # mark as done
         logging.info("Task complete")
         self.output().touch()
+
+
+class GridRootTask(luigi.WrapperTask):
```

Comment:
Added a upstream `RootTask` so I didn't need to run the full pipeline - just the tier-0 collection